test.spec.js

```diff
@@ -0,0 +1,47 @@
+const kijiji = require('./');
+jest.setTimeout(30000);
+
+describe('kijiji-scraper', () => {
+
+    describe('categories', () => {
+
+        it('should return you the kijiji category id for cars and vehicles', () => {
+            expect(kijiji.categories.CARS_AND_VEHICLES.id).toBe(27);
+        });
+
+    });
+
+    describe('locations', () => {
+
+        it('should be able to return the kijiji location id of Ottawa', () => {
+            expect(kijiji.locations.ONTARIO.OTTAWA_GATINEAU_AREA.OTTAWA.id).toBe(1700185);
+        });
+
+    });
+
+    describe('class search', () => {
+
+        describe('method search', () => {
+
+            let results;
+
+            beforeAll(async () => {
+                results = await kijiji.search({
```

Comment:
I like the idea of having tests that actually fetch Kijiji data, but I think there should be a distinction between unit tests (i.e., with Kijiji mocked out) and integration tests. Could you split this into two files, each in a `tests` directory - `tests/unitTests.spec.ts` and `tests/integrationTests.spec.ts`?

Reply:
Changed the tests into two different spec files under `./tests/`

`npm run test`
`npm run test-integration`